### PR TITLE
Updated Folder Structure, Deleted exe files, added BFS in Graph

### PR DESCRIPTION
**Pull request by anishjain94**
Updated Folder Structure to keep the repo more simple, Deleted exe files and test files and added BFS in Graph

### CODE REVIEW

File: Graph Theory/BFS.cpp [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+
+int main() 
+{ 
+	Graph g(4); 
+	g.addEdge(0, 1); 
+	g.addEdge(0, 2); 
+	g.addEdge(1, 2); 
+	g.addEdge(2, 0); 
+	g.addEdge(2, 3); 
+	g.addEdge(3, 3);
```
Comment (ekdnam) [inline Graph Theory/BFS.cpp]: Well done! Can this be done dynamically? WIth inputs from the user?

**Review by ekdnam — CHANGES_REQUESTED** [review]
Some changes are required, please look into them!

File: Graph Theory/BFS.cpp (hunk 1 above)
Comment (anishjain94) [reply Graph Theory/BFS.cpp]: Yep, it can be!

**Comment by ekdnam** [comment]
Why have you deleted the other files? Can't understand that

**Comment by anishjain94** [comment]
> Why have you deleted the other files? Can't understand that

I just deleted the exe files and the folders in which only a single program was there.

**Comment by ekdnam** [comment]
Okay. The BFS file should be in `Graph Theory/BFS`. And `Searching/Binary Search/README.md` should be deleted, as an identical exists here.

**Pull request closed without merging**